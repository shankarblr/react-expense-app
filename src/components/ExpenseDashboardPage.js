import React from 'react'
import ExportList  from './ExportList'
import ExpenseListFilter from './ExpenseListFilter'

const ExpenseDashboardPage = () =>(
    <div>
        <ExpenseListFilter />
        <ExportList />
    </div>
)

export default ExpenseDashboardPage