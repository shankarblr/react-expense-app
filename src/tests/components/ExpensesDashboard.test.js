import React from 'react'
import { shallow } from 'enzyme'
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage'

test('Test Expenses Dashboard compponent', () => {
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(wrapper).toMatchSnapshot();
});