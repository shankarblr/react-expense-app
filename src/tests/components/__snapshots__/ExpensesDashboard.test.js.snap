// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Expenses Dashboard compponent 1`] = `
<div>
  <Connect(ExpenseListFilter) />
  <Connect(ExportList) />
</div>
`;
